namelists/fesom2/2.5/awicm3/v3.2/namelist.io

```diff
@@ -35,8 +35,8 @@ io_list =  'sst       ',1, 'm', 4,
            'MLD3      ',1, 'm', 4,
            'tx_sur    ',1, 'm', 4,
            'ty_sur    ',1, 'm', 4,
-           'temp      ',1, 'y', 4,
-           'salt      ',1, 'y', 8,
+           'temp      ',1, 'm', 4,
```

Comment:
This change is intentional. Needed for CMIPtool. I had too many bad runs with only yearly output.